src/locales/en.json

```diff
@@ -137,68 +137,76 @@
+            "footer": "For more information, please see [the CC wiki's page on Considerations for Licensors and Licensees](https://wiki.creativecommons.org/wiki/Before_Licensing)."
         },
         "how-formally-license": {
             "heading": "How do I Formally License my Work?",
-            "text": "<p class='help-text'>Licensing your work is as simple as marking it under the specific license you choose. This marking can be as simple as a bit of text stating the license in a copyright notice, or as complex as embedding the license information on your website using the HTML code associated with the particular license. We strongly recommend at least including a link to the applicable license.</p>"
+            "text": "Licensing your work is as simple as marking it under the specific license you choose. This marking can be as simple as a bit of text stating the license in a copyright notice, or as complex as embedding the license information on your website using the HTML code associated with the particular license. We strongly recommend at least including a link to the applicable license.",
+            "footer": ""
         },
         "six-cc-licenses": {
             "heading": "The Six Creative Commons Licenses",
-            "text": "<p class='help-text'>There are six main licenses that Creative Commons offers.</p> <p class='help-text'>In the diagram to the right, you can see the six main licenses and the public domain CC0 license, and how restrictive they are, with licenses at the top being the least restrictive, and licenses at the bottom being the most restrictive.</p> <p class='help-text'>If you are unsure about which one is right for you and your work, please select <strong>\"I need help selecting a license\"</strong> on the first question of this page.</p>",
-            "footer": "For more information, please read more about  <a id='licensing_examples_link' href='https://creativecommons.org/share-your-work/licensing-types-examples/' target='_blank'>CC Licenses and Examples.</a>"
+            "text": "There are six main licenses that Creative Commons offers.\nIn the diagram to the right, you can see the six main licenses and the public domain CC0 license, and how restrictive they are, with licenses at the top being the least restrictive, and licenses at the bottom being the most restrictive.\nIf you are unsure about which one is right for you and your work, please select **I need help selecting a license** on the first question of this page.",
```

Comment:
Do you mean for this part: ` Creative Commons offers.\nIn the` instead of using the `\n` here? I've used it because `markdown-it` can add `<br>` in place of every `\n` in text, which I then convert into `</p><p>` to create paragraphs.

Reply:
I just meant that if you use template strings, you won't have to write the `\n` and can directly type newlines like this:
```js
const longText = `This is line 1
This is line 2`

longText === 'This is line 1\nThis is line 2' // true
```

Reply:
Oh, this is because it's a json file, json doesn't allow you to use template literals. I _could_ change it into a `.js` file, but is it worth it?